From: Chow Loong Jin <hyperair@debian.org>
Date: Sun, 21 Nov 2021 00:31:58 +0800
Subject: Stop building test_voronoi.cpp

test_voronoi.cpp compmilation hangs forever, see https://bugs.debian.org/998049.
Disable test_voronoi.cpp to fix the FTBFS until that is fixed.

Bug-Debian: https://bugs.debian.org/984338
---
 tests/libslic3r/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/libslic3r/CMakeLists.txt b/tests/libslic3r/CMakeLists.txt
index 69ae901..ab9b169 100644
--- a/tests/libslic3r/CMakeLists.txt
+++ b/tests/libslic3r/CMakeLists.txt
@@ -26,7 +26,6 @@ add_executable(${_TEST_NAME}_tests
 	test_region_expansion.cpp
 	test_timeutils.cpp
 	test_utils.cpp
-	test_voronoi.cpp
     test_optimizers.cpp
     test_png_io.cpp
     test_surface_mesh.cpp
